src/components/organisms/PageHeader/PreviewControl/PreviewLabel.tsx

```diff
@@ -22,7 +22,7 @@ export function PreviewLabel() {
       <LabelBox>
         {PREVIEW_ICON_MAP[preview.type]}
         <LabelContent>
-          Previewing <span style={{fontWeight: 700}}>{previewDisplayContent.name}</span>
+          Dry-running <span style={{fontWeight: 700}}>{previewDisplayContent.name}</span>
```

Comment:
"Performing Dry-run" ? or is that too long?

Reply:
oh - then "Viewing Kustomize dry run" is probably better - and "Viewing Helm-chart dry run" ? or "Showing.. " ? @CortNick wdyt? 

Also - grammatically it seems to be more correct to not hyphenise - i.e. "dry run" - but that looks even weirder!? And should we capitalize? i.e. "dry-run" or "Dry-run"?